multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -402,6 +403,7 @@ def __init__(
         self._sample_data_path = sample_data_path
         self._fit_called = False  # While using ddp, after fit called, we can only use single gpu.
         self._matcher = None
+        self._clean_old_ckpts = clean_old_ckpts
```

Comment:
Is it better to make clean_old_ckpts an argument in `fit()`? Whether to clean or not is a behavior of one training trial.

Reply:
Besides, `clean_old_ckpts` make users wonder what are old checkpoints. How about just calling `clean_ckpts`?

Reply:
Moved to fit and renamed to clean_ckpts.